packages/console/src/pages/SignInExperience/components/Welcome.tsx

```diff
@@ -9,7 +9,11 @@ import CardTitle from '@/components/CardTitle';
 import GuideModal from './GuideModal';
 import * as styles from './Welcome.module.scss';
 
-const Welcome = () => {
+type Props = {
+  mutate: () => void;
```

Comment:
nit: maybe `onClose`?

Reply:
But the component "Welcome" is not a modal, can not be closed, so `mutate` will be more clear.